RickAndMortyFunApp/NetworkServices/NetworkService.swift

```diff
@@ -6,56 +6,97 @@
 //  Copyright © 2019 Rusłan Chamski. All rights reserved.
 //
 
+import Alamofire
 import Foundation
 
 enum NetworkError: Error {
     case invalidURL
 }
 
+enum NetworkRouter: URLRequestConvertible {
+    static let baseURLString = "https://rickandmortyapi.com/api/character/"
+
+    case get(Int)
+    case create([String: Any])
+    case delete(Int)
+
+    func asURLRequest() throws -> URLRequest {
+        var method: HTTPMethod {
```

Comment:
I know where it came from, but let's move it to router level, because this function will become huge if we keep everything there. So, method, params, url and even encoding.